readthedocs/api/base.py

```diff
@@ -32,7 +32,7 @@
 class ProjectResource(ModelResource, SearchMixin):
     users = fields.ToManyField('readthedocs.api.base.UserResource', 'users')
 
-    class Meta:
+    class Meta(object):
```

Comment:
Also seems there are a bunch of other places where we're doing it and it isn't alerting. Is it special casing Django model's and forms, but not the api stuff?